docs/dev/Kibana-Reporting-Design-Proposal.md

```diff
@@ -55,13 +56,24 @@ Kibana Reports for Open Distro allows ‘Report Owner’ (engineers, including b
 1. As a user, I should be able to view, edit and delete a report definition.
 1. As a user, I should be able to enable/disable a report definition
 
+
+#### Search module
```

Comment:
please move this section to `create report definition` section, because the search module should be in the create report page, not the report landing page

Reply:
ok